lua/header.lua

```diff
@@ -160,7 +167,40 @@ local function add_license_header(opts)
         local commented_headers = comment_headers(license_table, comments)
         vim.api.nvim_buf_set_lines(buffer, 0, 0, false, commented_headers)
     else
-        print("Unsupported file type:", file_extension)
+        vim.notify_once("unsupported file type for adding header", vim.log.levels.ERROR)
+    end
+end
+
+function header.update_date_modified()
```

Comment:
```suggestion
local function header.update_date_modified()
```
it is required to be local and at the of file you can do:
```
header.update_date_modified = function()
    update_date_modified()
end
```
so that the update method can be reached from outside